packages/scripts/src/docker-build/lib.ts

```diff
@@ -80,7 +80,7 @@ async function makeDockerComposeFile(
   )
 
   return {
-    version: '3.9',
+    version: '3.3',
```

Comment:
Should version be upgraded now?

Reply:
I moved the build jobs to our self hosted instances to speed them up, and the docker version in there is not as new as the ubuntu-latest one. It should have no effect as we don't use any newer features